lib/classes/plugininfo/qbank.php

```diff
@@ -130,4 +130,33 @@ public function load_settings(\part_of_admin_tree $adminroot, $parentnodename, $
             $ADMIN->add($parentnodename, $settings);
         }
     }
+
+    /**
+     * Pre-uninstall hook.
+     *
+     * This is intended for disabling of plugin, some DB table purging, etc.
+     *
+     * NOTE: to be called from uninstall_plugin() only.
+     * @private
+     */
+    public function uninstall_cleanup() {
+        $plugintoremove = $this->displayname;
+        self::remove_unused_column_from_db($plugintoremove);
+    }
```

Comment:
Please call parent::pre_uninstall_cleanup() at the end `parent::uninstall_cleanup();`

Reply:
Hi @guillogo looks like the method `pre_uninstall_cleanup()` does not exist in the parent class `lib/classes/plugininfo/base.php` should I call `uninstall_cleanup` instead ?